index.html

```diff
@@ -57,22 +57,16 @@
                 <hr>
                 <p id="civtext"></p>
             </div>
-            <div id="version" class="info">
-                Version: Age of Empires II DE Update <a href="https://www.ageofempires.com/news/aoe2de-update-35584/" target="_blank" title="Changelog">35584</a>
-            </div>
-            <br/>
-            <div id="github" class="info">
-                Contribute on <a href="https://github.com/SiegeEngineers/aoe2techtree" target="_blank">GitHub</a>
-            </div>
-            <br/>
-            <div id="thanks" class="small_info">
+            <div class="info">
                 <div>
-                    Made by hszemi, Anda, exterkamp, paulirish,
+                    Version: Age of Empires II DE Update <a href="https://www.ageofempires.com/news/aoe2de-update-35584/" target="_blank" title="Changelog">35584</a>
                 </div>
                 <div>
-                    with thanks to Jineapple, TriRem, pip, and NkoDragaš
+                    Contribute on <a href="https://github.com/SiegeEngineers/aoe2techtree" target="_blank">GitHub</a>
                 </div>
                 <div>
+                    Made by hszemi, Anda, exterkamp, paulirish,</br>
+                    with thanks to Jineapple, TriRem, pip, and NkoDragaš</br>
```

Comment:
Isn't it `<br/>` (or just `<br>`), instead of `</br>`?

Reply:
Good catch!